app/uk/gov/hmrc/nationalinsurancerecord/controllers/auth/AuthAction.scala

```diff
@@ -77,13 +75,12 @@ class AuthActionImpl @Inject()(val authConnector: AuthConnector)(implicit execut
 }
 
 @ImplementedBy(classOf[AuthActionImpl])
-trait AuthAction extends ActionBuilder[Request] with ActionFilter[Request]
+trait AuthAction extends ActionBuilder[Request, AnyContent] with ActionFilter[Request]
 
-class AuthConnector @Inject()(val http: WSHttp,
-                              val runModeConfiguration: Configuration,
-                              environment: Environment
-                             ) extends PlayAuthConnector with ServicesConfig {
-  override val serviceUrl: String = baseUrl("auth")
+class AuthConnector @Inject()(val http: HttpClient,
```

Comment:
AuthConnector comes from bootstrap now so this can be removed.

Reply:
Done